src/pages/integrations/bitpay-card/bitpay-card-home/bitpay-card-home.ts

```diff
@@ -89,7 +96,51 @@ export class BitPayCardHome implements OnInit {
     return index;
   }
 
+  public async runCardAudienceEvents() {
+    let cards = await this.persistenceProvider.getBitpayDebitCards('livenet');
```

Comment:
Can we wrap everything is this method with a try catch? probably not needed but it can't hurt.